packages/runtime/runtime-utils/src/summarizerNode/summarizerNode.ts

```diff
@@ -217,16 +217,26 @@ export class SummarizerNode implements IRootSummarizerNode {
     public async refreshLatestSummary(
         proposalHandle: string | undefined,
         summaryRefSeq: number,
-        getSnapshot: () => Promise<ISnapshotTree>,
+        fetchLatestSnapshot: () => Promise<IFetchSnapshotResult>,
         readAndParseBlob: ReadAndParseBlob,
         correlatedSummaryLogger: ITelemetryLogger,
     ): Promise<RefreshSummaryResult> {
+        this.defaultLogger.sendTelemetryEvent({
```

Comment:
@agarwal-navin Should I remove this or is it useful to have?  Not part of the original PR (some other change)

Reply:
It's useful to have.